packages/headless/src/features/commerce/context/cart/cart-selector.ts

```diff
@@ -0,0 +1,32 @@
+import {Ec} from '@coveo/relay-event-types';
+import {createSelector} from '@reduxjs/toolkit';
+import {CommerceEngineState} from '../../../../app/commerce-engine/commerce-engine';
+import {getCurrency} from '../context-selector';
+import {CartState} from './cart-state';
+
+export interface Transaction {
```

Comment:
Nit: just to mention that a [Transaction](https://github.com/coveo/ui-kit/pull/3654/files#diff-64e4398703c6034135ce88296c4b4bb23fcb6c55b19eee73655ef0b228a1383cR55) interface is declared in the headless-cart file at the moment (it might be the same purpose of this one).